tests/test_benchmark_simple_scenarios.py

```diff
@@ -261,6 +258,50 @@ def test_benchmark_AE_grid_battery_peak_pricing(self, margs):
                 ):
                     assert abs(battery_charge[i - 1]) > 0
 
+    @pytest.mark.skipif(
+        EXECUTE_TESTS_ON not in (TESTS_ON_MASTER),
+        reason="Benchmark test deactivated, set env variable "
+        "EXECUTE_TESTS_ON to 'master' to run this test",
+    )
+    @mock.patch("argparse.ArgumentParser.parse_args", return_value=argparse.Namespace())
+    def test_benchmark_AFG_grid_heatpump_heat(self, margs):
+        use_case = "AFG_grid_heatpump_heat"
```

Comment:
To me, the titles of the benchmark tests become more and more cryptic. We need a descrition below `def ...:` to know what the test is about. 

I would add something like this:
```suggestion
    def test_benchmark_AFG_grid_heatpump_heat(self, margs):
        r"""
        Benchmark test for a sector-coupled energy system, including electricity and heat demand. As an sector-coupling asset, a heat pump is used. Both an electricity and heat DSO are present. The electricity tariff is defined as a timeseries. When...
        """
        use_case = "AFG_grid_heatpump_heat"
```